### ISSUE TITLE
[1.34.0] ReactAppboy.getInitialURL is never resolved / rejected on iOS when there is no initial URL

### ISSUE THREAD

**Issue by mlazari** [Appboy/appboy-react-sdk#140]
After updating from `1.33.1` to `1.34.0` `ReactAppboy.getInitialURL` is never resolved / rejected because of this change: https://github.com/Appboy/appboy-react-sdk/commit/38c2e256eaeedadf27b046e053b748bd61352456#diff-b9c42b21b836c8e301ed879300de21f53f4d96193298b98128e66cd186659a81L85

**Comment by Bucimis** [comment]
@mlazari does replacing the `getInitialUrl` code in `AppboyReactBridge.m` with the following:

```
RCT_EXPORT_METHOD(getInitialUrl:(RCTResponseSenderBlock)callback) {
  if ([AppboyReactUtils sharedInstance].initialUrlString != nil) {
    [self reportResultWithCallback:callback andError:nil andResult:[AppboyReactUtils sharedInstance].initialUrlString];
  } else {
    [self reportResultWithCallback:callback andError:nil andResult:nil];
  }
}
```

fix your issue? If so we'll look into releasing that as a fix

**Comment by mlazari** [comment]
@Bucimis Yes, replacing with that code fixes the issue. Thank you for looking into this!

**Comment by matinzd** [comment]
Broken from this commit, promise will never resolve nor reject:
https://github.com/Appboy/appboy-react-sdk/commit/38c2e256eaeedadf27b046e053b748bd61352456

**Comment by wesleyorbin** [comment]
We just released 1.34.1 with this fix. Feel free to re-open if the issue persists.
